<!-- sobit_follower.launch -->
<launch>

	<!-- velocity smoother -->
	<include file="$(find sobit_follower)/launch/velocity_smoother_2.launch"/>
    <!-- Points_grouping -->
    <node pkg="sobit_follower" type="Points_grouping_URG" name="Points_grouping_node" output="screen"/>
    <!-- target_scan -->
    <node pkg="sobit_follower" type="target_scan_check" name="target_scan_node" output="screen"/>
	<!-- path_plan -->
	<node pkg="sobit_follower" type="path_plan_test_3" name="route_planning_node" output="screen"/>
	<!-- kalman -->
	<node pkg="sobit_follower" type="kalman_check2" name="kalman_check2_node" output="screen"/>
	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find sobit_follower)/rviz/test.rviz"/>
	<!-- txt -->
	<!--node pkg="sobit_follower" type="file_wr.py" name="point_listener" output="screen"/-->
</launch>
